.circleci/config.yml

```diff
@@ -0,0 +1,65 @@
+version: 2
+jobs:
+  build-and-test:
+    working_directory: /ITKSubdivisionQuadEdgeMeshFilter-build
+    docker:
+      - image: insighttoolkit/module-ci:latest
+    steps:
+      - checkout:
+          path: /ITKSubdivisionQuadEdgeMeshFilter
+      - run:
+          name: Fetch CTest driver script
+          command: |
+            curl -L https://raw.githubusercontent.com/InsightSoftwareConsortium/ITK/dashboard/itk_common.cmake -O
+      - run:
+          name: Configure CTest script
+          command: |
+            SHASNIP=$(echo $CIRCLE_SHA1 | cut -c1-7)
+            cat > dashboard.cmake << EOF
+            set(CTEST_SITE "CircleCI")
+            set(CTEST_BUILD_NAME "External-ITKSubdivisionQuadEdgeMeshFilter-${CIRCLE_BRANCH}-${CIRCLE_BUILD_NUM}-${SHASNIP}")
+            set(CTEST_BUILD_CONFIGURATION "MinSizeRel")
+            set(CTEST_CMAKE_GENERATOR "Unix Makefiles")
+            set(CTEST_BUILD_FLAGS: "-j5")
+            set(CTEST_SOURCE_DIRECTORY /ITKSubdivisionQuadEdgeMeshFilter)
+            set(CTEST_BINARY_DIRECTORY /ITKSubdivisionQuadEdgeMeshFilter-build)
+            set(dashboard_model Experimental)
+            set(dashboard_no_clean 1)
+            set(dashboard_cache "
+            ITK_DIR:PATH=/ITK-build
+            BUILD_TESTING:BOOL=ON
+            ")
+            include(\${CTEST_SCRIPT_DIRECTORY}/itk_common.cmake)
+            EOF
+      - run:
+          name: Build and Test
+          no_output_timeout: 1.0h
+          command: |
+            ctest -j 2 -VV -S dashboard.cmake
+  package:
+    working_directory: ~/
+    machine: true
+    steps:
+      - checkout:
+          path: ~/ITKSubdivisionQuadEdgeMeshFilter
```

Comment:
@jhlegarreta This path may be interfering with the execution of `./dockcross-manylinux-download-cache-and-build-module-wheels.sh` and can likely be removed, per

https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/blob/965dcc71ae1179bc254e9e34f12930add4798f4e/%7B%7Bcookiecutter.project_name%7D%7D/.circleci/config.yml#L43

Reply:
@thewtex 👍 Thanks. Done in 59fb4d4.